featurebyte/schema/target_table.py

```diff
@@ -8,24 +8,15 @@
 
 from pydantic import Field, StrictStr, root_validator
 
-from featurebyte.models.base import FeatureByteBaseModel, PydanticObjectId
-from featurebyte.models.request_input import RequestInputType
-from featurebyte.models.target_table import TargetTableModel
+from featurebyte.models.base import PydanticObjectId
+from featurebyte.models.observation_table import ObservationTableModel, TargetInput
 from featurebyte.query_graph.graph import QueryGraph
 from featurebyte.query_graph.node import Node
 from featurebyte.schema.common.base import PaginationMixin
 from featurebyte.schema.common.feature_or_target import FeatureOrTargetTableCreate
 from featurebyte.schema.materialized_table import BaseMaterializedTableListRecord
 
 
-class ObservationInputType(FeatureByteBaseModel):
```

Comment:
Removed this class since we can infer the input type in the backend based on whether `observation_table_id` is None